pkg/systemd/services.html

```diff
@@ -122,6 +122,23 @@
             {{/TemplateDescription}}
           </table>
         </div>
+        {{#NotMetConditions.length}}
+        <div class="list-group-item">
+          <table>
+            <tr>
+              <td>
+                <span>condition failed</span>
```

Comment:
Lets ask a designer: @garrett Could you tell us if Condition failed should be capitalized?

@marusak If @garrett doesn't have time to reply (he's travelling) and you get everything else fixed up on this pull request (such as tests) ... then I would say: capitalize it.